config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -477,12 +477,16 @@ periodics:
             cpu: 2
             memory: "9Gi"
         env:
+          - name: API_SERVER_FEATURE_GATES
+            value: "InPlacePodVerticalScaling=true"
           - name: GINKGO_FOCUS
-            value: (\[Feature:NodeLogQuery\]|\[sig-windows\]) # run the features and a few windows related tests
+            value: (\[Feature:InPlacePodVerticalScaling\]|\[Feature:NodeLogQuery\]|\[sig-windows\]) # run the features and a few windows related tests
           - name: NODE_FEATURE_GATES
-            value: "NodeLogQuery=true"
-          - name: WINDOWS_SERVER_VERSION
-            value: "windows-2022"
+            value: "InPlacePodVerticalScaling=true,NodeLogQuery=true,RuntimeClassInImageCriApi"
```

Comment:
Shouldn't it be `RuntimeClassInImageCriApi=true`?

Reply:
can we add PodAndContainerStatsFromCRI=true

Reply:
https://github.com/kubernetes/test-infra/blob/650a3943d3e33a28eda27d0d686b1e3dda2437b9/config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml#L166

Reply:
whoops, yes

Reply:
updated (added the =true fo the RUntimeClassINIMageCriApi and added the PodAndContainerStatsFromCRI=true feature gates)